docs/sources/azureblobstorage.md

```diff
@@ -0,0 +1,158 @@
+![Custom IAM role](../images/azureblobstorage-source/iam-2.png)
+
+The corresponding role JSON is given as a reference which you can replicate to create a similar custom IAM role:
+
+```json
+{
+    "properties": {
+        "roleName": "TriggerMesh Event Grid subscriptions manager",
+        "description": "Allows the usage of TriggerMesh event sources for Azure Blob Storage.",
+        "assignableScopes": [
+            "/subscriptions/15537daf-e607-4df8-b2ef-277248b205b3"
+        ],
+        "permissions": [
+            {
+                "actions": [
+                    "Microsoft.EventGrid/eventSubscriptions/read",
+                    "Microsoft.EventGrid/eventSubscriptions/write",
+                    "Microsoft.EventGrid/eventSubscriptions/delete",
+                    "Microsoft.EventHub/namespaces/eventhubs/write"
+                ],
+                "notActions": [],
+                "dataActions": [],
+                "notDataActions": []
+            }
+        ]
+    }
+}
+```
+
+After the Service Principal is created and assigned suitable roles, take note of the following information:
+
+* **Tenant ID** and **Client ID** (see _Get tenant and app ID values for signing in_)
+* **Client secret** (see _Create a new application secret_)
+
+### Event Hubs Namespace
+
+Follow the instructions at [Quickstart: Create an Event Hub using Azure portal][eventhubs-create], and create a new
+Event Hubs namespace that will contain an Event Hubs instance which we will use as the destination of events originating
+from the Azure Storage Account.
+
+![Event Hubs namespaces](../images/azureblobstorage-source/eventhubs-ns-1.png)
+
+### Event Hubs Instance _(optional)_
+
+This section can be skipped if you would like to let the event source manage its own Event Hub, which it sets as the
+destination of events originating from the Azure Storage Account. In this case, please ensure you granted all necessary
+permissions to the Service Principal in the previous section.
+
+If, however, you prefer to provide your own Event Hub for that purpose, follow the instructions at [Quickstart: Create
+an Event Hub using Azure portal][eventhubs-create] to create an Event Hubs instance. Take note of its resource ID, it
+will be used to run an instance of the Azure Blob Storage event source.
+
+A resource ID for an Event Hub has the format:
+
+```
+/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.EventHub/namespaces/{namespaceName}/eventHubs/{eventHubName}
+```
+
+![Event Hub overview](../images/azureblobstorage-source/eventhub-1.png)
+![Event Hub resource ID](../images/azureblobstorage-source/eventhub-2.png)
+
+Resource IDs can also be obtained using the [Azure CLI][azure-cli] (`az`). The following command line uses values from
+the screenshots above:
+
+```console
+$ az eventhubs eventhub show --resource-group blobstorage-source-dev --namespace-name eventsourcedev --name my-event-hub
+TODO
```

Comment:
why is there a `TODO` here ?

Reply:
Please wait, the PR is a draft. We don't have a form yet so there will be things missing.